test/Atmos/EDMF/closures/turbulence_functions.jl

```diff
@@ -145,6 +145,12 @@ Returns the turbulent Prandtl number, given:
  - `Grad_Ri`, the gradient Richardson number
 """
 function turbulent_Prandtl_number(Pr_n::FT, Grad_Ri::FT, ω_pr::FT) where {FT}
-    denom = 1 + ω_pr * Grad_Ri - sqrt((1 + ω_pr * Grad_Ri)^2 - 4 * Grad_Ri)
-    return Pr_n * 2 * Grad_Ri / denom
+    if Grad_Ri > FT(0)
+        factor =
+            2 * Grad_Ri /
+            (1 + ω_pr * Grad_Ri - sqrt((1 + ω_pr * Grad_Ri)^2 - 4 * Grad_Ri))
+    else
+        factor = FT(1)
+    end
```

Comment:
Maybe we can start with the other PR, to demonstrate that this is a continuous function.

Reply:
@charleskawczynski yair told me about the intent to make a plot for this. Can we make a unit test instead, evaluating it for values from -1 to 1 for example?

Reply:
That would be nice, but I think plotting this would also be useful because 1) the function appears in the denominator so we really need to be sure it's robust, and 2) this function is not obviously continuous. It took me some time fixing the universal functions, which also had tests and several if switches and non-defined regions, once I plotted them to validate them against literature.